README.md

```diff
@@ -20,7 +20,7 @@ For more details see [license.txt](https://github.com/LowPowerLab/RaspberryPi-Ga
 http://lowpowerlab.com/gateway
 
 ###Quick reference:
-- Copy index.php, graphHelper.js and images, sounds subfolders in your /var/www/default/ folder.
```

Comment:
The point was by removing the php requirement it should be changed to html. Looks like when I was reworking my patch for submission I left out the file rename though.